week-3/02-course-app-easy/index.js

```diff
@@ -7,48 +8,144 @@ let ADMINS = [];
 let USERS = [];
 let COURSES = [];
 
+let id = 1;
+
 // Admin routes
-app.post('/admin/signup', (req, res) => {
-  // logic to sign up admin
+app.post('/admin/signup', (req, res) => {// logic to sign up admin
+  let admin = req.body;
```

Comment:
you should write logic if admin is already registered

Reply:
Yep. I thought that. But if we go like that many validations we have to write. So I just kept it simple.
For Example: Logic for checking whether course is already purchased before purchasing it.

Reply:
> you should write logic if admin is already registered

As you suggested I have done the changes, committed and pushed it.
Added validation while signup for Admin and User. Also added validations for all other activities of Admin and User.

One Doubt:
Is Pushing the changes is fine or I have to create a new PR. Because I am unable to create a PR for second commit.

Reply:
I talked to one TA, he told no need to raise PR actually.
just do it and push in our branch.

Reply:
Okay. Did you check the validation logics. Are they fine?